content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+2. **Set white and black points**: The next two steps use the [_filmic rgb_](../../module-reference/processing-modules/filmic-rgb.md) module to define how the tones in your image will be mapped to the dynamic range of your display. Start by setting the _white_ and _black_ relative exposure sliders in the [_scene tab_](../../module-reference/processing-modules/filmic-rgb.md#scene). These are purely technical settings, defining white and black relative to the mid-gray point you set in the previous step. If your image contains tones you want to treat as pure white or pure black you can use the color pickers beside the sliders to set these values (using the maximum and minimum brightness of the image). Otherwise set the values manually using the color assessment frames as a reference.
+
+3. **Adjust the contrast**: Now move to the [_look tab_](../../module-reference/processing-modules/filmic-rgb.md#look) in _filmic rgb_ (for now we will skip the [_reconstruct tab_](../../module-reference/processing-modules/filmic-rgb#reconstruct)). Enable the [_look only_ view](../../module-reference/processing-modules/filmic-rgb.md#graphic-display) at the top of the module to see a representation of the filmic tone curve, which consists of a straight section in the middle (used to set the contrast of the mid-tones) and curved sections at the top and bottom (where the shadows and highlights are compressed to fit the dynamic range of the display). 
+
+   The _contrast_ slider changes the slope of the straight section (the mid-tone image contrast), the _latitude_ slider changes its length and the _shadows/highlights balance_ slider changes its position. There is a lot of give-and-take involved here -- if you want to increase the contrast of the mid-tones, you must sacrifice contrast in the shadows/highlights and vice versa. Experiment with these sliders to understand how they affect the image.
+   
+---
+
+  Note: The highlight compression in the filmic rgb module can cause detail to be lost in the highlights. You can mitigate this to some extent by reducing the _white relative exposure_, adjusting the _shadows/highlights balance_ or changing the _contrast in highlights_ setting in the [_options tab_](../../module-reference/processing-modules/filmic-rgb.md#options). The [_tone equalizer_](../../module-reference/processing-modules/tone-equalizer.md) module can also be used to reduce the relative brightness of the sky.
+
+---
+
+4. **Color preservation**: The tone mapping in the filmic module attempts to redistribute the tones in your image without affecting color reproduction. While the default color preservation algorithm works for most images, you are encouraged to experiment by changing the _preserve chrominance_ setting in the [_options tab_](../../module-reference/processing-modules/filmic-rgb.md#options) if you do not like how the colors appear. 
```

Comment:
This needs more explanation, although maybe not here. Can we link to some other section to provide more detail?

Reply:
I already link to the filmic documentation. If more information is needed it should go there.